packages/navitem/src/react/index.tsx

```diff
@@ -24,12 +26,22 @@ const NavItem: React.FC<NavItemProps> & NavItemStatics = props => {
     children,
     icon,
     menu = initialValue.menu,
-    renderContainer = initialValue.renderContainer,
-    selected = initialValue.selected
+    renderContent = initialValue.renderContent,
```

Comment:
There's a weirdness in setting prop defaults with context defaults and then setting the context value with the result.

But I think I know why you're doing it: probably so you have fewer null checks to do on menu, renderContent and selected w/in the codebase.

Good news is that menu and selected are booleans, so those can easily go w/o a default.

Alignment is similar because it's "either-or" vert/horz value and could have a default/fallback.

This might mean you don't have to ensure non-nulls after all.